lnrpc/walletrpc/walletkit.proto

```diff
@@ -400,6 +407,19 @@ message ListAccountsResponse {
     repeated Account accounts = 1;
 }
 
+message RequiredReserveRequest {
+    // The number of additional channels the user would like to open.
+    uint32 additional_public_channels = 1;
+}
+
+message RequiredReserveResponse {
+    // The amount of reserve required.
+    int64 required_reserve = 1;
+
+    // The balance of the wallet.
+    int64 wallet_balance = 2;
```

Comment:
I honestly also don't see the point of duplicating the wallet balance again here. I think we should remove it to have a concise API.